packages/appeals-service-api/src/routes/v2/users/_id/appeal-submissions/_appealSubmissionId/service.js

```diff
@@ -0,0 +1,36 @@
+const { UserAppealSubmissionRepository } = require('./repo');
+const { AppealsRepository } = require('#repositories/appeals-repository');
+
+const repo = new UserAppealSubmissionRepository();
+const cosmosAppeals = new AppealsRepository();
+
+/**
+ * @typedef {import("@prisma/client").AppealCase} AppealCase
+ * // TODO: define type for submission (ideally generated from spec)
```

Comment:
We have this: `import('appeals-service-api').Api.AppealSubmission` - but we don't have a DB type for it, as it's in Cosmos wasn't documented.

Reply:
Nice, swapped to the imported type, thanks!